mne/beamformer/tests/test_dics.py

```diff
@@ -44,7 +44,7 @@ def _get_data(tmin=-0.11, tmax=0.15, read_all_forward=True, compute_csds=True):
     if read_all_forward:
         forward_surf_ori = _read_forward_solution_meg(fname_fwd, surf_ori=True)
         forward_fixed = _read_forward_solution_meg(fname_fwd, force_fixed=True,
-                                                   surf_ori=True)
+                                                   use_cps=False)
```

Comment:
with `force_fixed=True` the `surf_ori` parameter is ignored, so I deleted it. `use_cps=False` mimics the old behavior of `convert_forward_solution`, so the tests pass without changing any test.